hack/upgrade-test-operator-sdk.sh

```diff
@@ -314,14 +311,12 @@ VIRTIOWIN_IMAGE_CM=$(${CMD} get cm virtio-win -n ${HCO_NAMESPACE} -o jsonpath='{
 
 [[ "${VIRTIOWIN_IMAGE_CSV}" == "${VIRTIOWIN_IMAGE_CM}" ]]
 
-Msg "Read the HCO operator log before it been deleted"
+Msg "Read the HCO operator and webhook logs, before it been deleted"
 LOG_DIR="${ARTIFACT_DIR}/logs"
 mkdir -p "${LOG_DIR}"
-HCO_POD=$( ${CMD} get -n ${HCO_NAMESPACE} pods -l "name=hyperconverged-cluster-operator" -o name)
-${CMD} logs -n ${HCO_NAMESPACE} "${HCO_POD}" > "${LOG_DIR}/hyperconverged-cluster-operator.log"
-
-Msg "Read the HCO webhook log before it been deleted"
-WH_POD=$( ${CMD} get -n ${HCO_NAMESPACE} pods -l "name=hyperconverged-cluster-webhook" -o name)
-${CMD} logs -n ${HCO_NAMESPACE} "${WH_POD}" > "${LOG_DIR}/hyperconverged-cluster-webhook.log"
+for pod in $(${CMD} get pod -n ${HCO_NAMESPACE} -l "name in (hyperconverged-cluster-operator,hyperconverged-cluster-webhook)" -o custom-columns=name:.metadata.name --no-headers); do
```

Comment:
this would be shorter:
```suggestion
for pod in $(${CMD} get pod -n ${HCO_NAMESPACE} -l "name in (hyperconverged-cluster-operator,hyperconverged-cluster-webhook)" -o jsonpath='{.items[*].metadata.name}'; do
```